ddtrace/contrib/rq/__init__.py

```diff
@@ -202,7 +217,7 @@ def traced_perform_job(rq, pin, func, instance, args, kwargs):
     finally:
         # Force flush to agent since the process `os.exit()`s
         # immediately after this method returns
-        pin.tracer.flush()
+        ctx["pin"].tracer.flush()
```

Comment:
This change looks incorrect to me given that `ctx` doesn't exist anymore at this point in the code.